nominees/notesmaster-.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "NotesMaster ",
+  "aliases": [
+    "Always learning, Together"
+  ],
+  "description": "Working in partnerships to support the sustainable access to digital learning resources.",
+  "website": "https://notesmaster.com/",
+  "license": [
+    {
+      "spdx": "CC-BY-SA-4.0",
+      "licenseURL": "https://notesmaster.com/termsofuse"
+    }
+  ],
+  "SDGs": [
+    {
+      "SDGNumber": 4,
+      "evidenceText": "As a freely accessible elearning platform, NotesMaster provides access to quality lessons in 31 countries, authored by local teachers and based around the local curriculum. The latest program to develop local digital skills and improve access to local content was in Trinidad and Tobago."
```

Comment:
Note that the e-learning platform (software) is different from the learning resources (content). If the argument is made here about the e-learning platform, the argument is not valid because the platform is not a digital public good.